build.sh

```diff
@@ -1,14 +1,22 @@
 #!/bin/sh
 
-set -eu
+set -eux
 
-ICE_VERSION=${1:-"3.6.4"}
+ICE_VERSION=${1:-"3.6.5"}
 
-pip download "zeroc-ice==$ICE_VERSION"
+# Build ice cpp from source
+wget -q https://github.com/zeroc-ice/ice/archive/v$ICE_VERSION.tar.gz
+tar xzf v$ICE_VERSION.tar.gz
+cd ice-$ICE_VERSION/cpp
+
+make --silent && make install --silent
+
+tar cf - /opt/Ice-$ICE_VERSION/ | xz -z - > /dist/Ice-$ICE_VERSION-ubuntu1804-amd64.tar.xz
```

Comment:
Might be better to switch to `gz` instead of `xz` to avoid installing another dependency to unarchive?